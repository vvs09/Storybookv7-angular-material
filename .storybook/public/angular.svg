<svg width="188" height="200" viewBox="0 0 188 200" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M94 123.4H69.3L57.6 152.6H35.9H35.8L94 22.1V22.2V0L0.899994 33.2L15.1 156.3L94 200V123.4Z" fill="#808080"/>
<path d="M77 105.4H94V64.5L77 105.4Z" fill="#808080"/>
<g opacity="0.9">
<path d="M94 123.4H118.7L130.4 152.6H152.1H152.2L94 22.1V22.2V0L187.1 33.2L172.9 156.3L94 200V123.4Z" fill="#808080"/>
<path d="M111 105.4H94V64.5L111 105.4Z" fill="#808080"/>
</g>
</svg>
