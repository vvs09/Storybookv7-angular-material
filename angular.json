{
  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
  "version": 1,
  "newProjectRoot": "projects",
  "projects": {
    "storybook": {
      "projectType": "application",
      "schematics": {
        "@schematics/angular:component": {
          "style": "sass"
        }
      },
      "root": "",
      "sourceRoot": "src",
      "prefix": "app",
      "architect": {
        "build": {
          "builder": "@angular-devkit/build-angular:browser",
          "options": {
            "outputPath": "dist/storybook",
            "index": "src/index.html",
            "main": "src/main.ts",
            "polyfills": [
              "zone.js"
            ],
            "tsConfig": "tsconfig.app.json",
            "inlineStyleLanguage": "sass",
            "assets": [
              "src/favicon.ico",
              "src/assets"
            ],
            "styles": [
              "@angular/material/prebuilt-themes/indigo-pink.css",
              "src/styles.sass"
            ],
            "scripts": []
          },
          "configurations": {
            "production": {
              "budgets": [
                {
                  "type": "initial",
                  "maximumWarning": "500kb",
                  "maximumError": "1mb"
                },
                {
                  "type": "anyComponentStyle",
                  "maximumWarning": "2kb",
                  "maximumError": "4kb"
                }
              ],
              "outputHashing": "all"
            },
            "development": {
              "buildOptimizer": false,
              "optimization": false,
              "vendorChunk": true,
              "extractLicenses": false,
              "sourceMap": true,
              "namedChunks": true
            }
          },
          "defaultConfiguration": "production"
        },
        "serve": {
          "builder": "@angular-devkit/build-angular:dev-server",
          "configurations": {
            "production": {
              "buildTarget": "storybook:build:production"
            },
            "development": {
              "buildTarget": "storybook:build:development"
            }
          },
          "defaultConfiguration": "development"
        },
        "extract-i18n": {
          "builder": "@angular-devkit/build-angular:extract-i18n",
          "options": {
            "buildTarget": "storybook:build"
          }
        },
        "test": {
          "builder": "@angular-devkit/build-angular:karma",
          "options": {
            "polyfills": [
              "zone.js",
              "zone.js/testing"
            ],
            "tsConfig": "tsconfig.spec.json",
            "inlineStyleLanguage": "sass",
            "assets": [
              "src/favicon.ico",
              "src/assets"
            ],
            "styles": [
              "@angular/material/prebuilt-themes/indigo-pink.css",
              "src/styles.sass"
            ],
            "scripts": []
          }
        },
        "storybook": {
          "builder": "@storybook/angular:start-storybook",
          "options": {
            "configDir": ".storybook",
            "browserTarget": "storybook:build",
            "compodoc": false,
            "port": 6006,
            "enableProdMode":true
          }
        },
        "build-storybook": {
          "builder": "@storybook/angular:build-storybook",
          "options": {
            "configDir": ".storybook",
            "browserTarget": "storybook:build",
            "compodoc": false,
            "outputDir": "storybook-static"
          }
        }
      }
    }
  }
}