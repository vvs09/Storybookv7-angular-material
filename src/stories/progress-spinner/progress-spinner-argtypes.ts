export const progressSpinnerargTypes = {
  color: {
    control: 'radio',
    options: ['accent', 'primary', 'warn'],
    description: 'Theme color palette for the component.',
    table: {
      category: 'INPUTS',
      defaultValue: { summary: '-' },
      type: {
        summary: 'string',
      },
    },
  },
  diameter: {
    control: 'number',
    description:
      'The diameter of the progress spinner (will set width and height of svg).',
    table: {
      category: 'INPUTS',
      defaultValue: { summary: '-' },
      type: {
        summary: 'number',
      },
    },
  },
  mode: {
    control: 'radio',
    options: ['determinate', 'indeterminate', 'buffer', 'query'],
    description:
      'Mode of the progress bar.Input must be one of these values: determinate, indeterminate, buffer, query, defaults to determinate.Mirrored to mode attribute.',
    table: {
      category: 'INPUTS',
      defaultValue: { summary: 'determinate' },
      type: {
        summary: 'string',
      },
    },
  },
  strokeWidth: {
    control: 'number',
    description: 'Stroke width of the progress spinner.',
    table: {
      category: 'INPUTS',
      defaultValue: { summary: '-' },
      type: {
        summary: 'number',
      },
    },
  },
  value: {
    control: 'number',
    description:
      'Value of the progress bar. Defaults to zero. Mirrored to aria-valuenow.',
    table: {
      category: 'INPUTS',
      defaultValue: { summary: '-' },
      type: {
        summary: 'number',
      },
    },
  },
};
