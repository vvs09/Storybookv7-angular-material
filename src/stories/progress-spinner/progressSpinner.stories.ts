import type { Meta, StoryObj } from '@storybook/angular';
import { argsToTemplate, moduleMetadata } from '@storybook/angular';
import {
  MatProgressSpinnerModule,
  MatProgressSpinner,
} from '@angular/material/progress-spinner';
import { progressBarargTypes } from '../progress-bar/progress-bar-argtypes';

const meta = {
  title: 'Components/Progress Spinner',
  component: MatProgressSpinner,
  tags: ['autodocs'],
  argTypes: {
    ...progressBarargTypes,
  },
  decorators: [
    moduleMetadata({
      imports: [MatProgressSpinnerModule],
    }),
  ],
  render: (args:any) => ({
    props: args,
    template: `<mat-progress-spinner aria-label="spinner" ${argsToTemplate(
      args
    )}> </mat-progress-spinner>
    `,
  }),
};

export default meta;
type Story = StoryObj<MatProgressSpinner>;

export const Default: Story = {
  args: {
    color: 'primary',
    mode: 'indeterminate',
    diameter: 80,
    strokeWidth: 5,
  },
};

export const Warn: Story = {
  args: {
    ...Default.args,
    color: 'warn',
  },
};

export const Accent: Story = {
  args: {
    ...Default.args,
    color: 'accent',
  },
};

export const StrokeWidth: Story = {
  args: {
    ...Default.args,
    strokeWidth: 10,
  },
};

export const Diameter: Story = {
  args: {
    ...Default.args,
    strokeWidth: 10,
    diameter: 100,
  },
};
