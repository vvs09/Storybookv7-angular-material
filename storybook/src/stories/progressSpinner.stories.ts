import type { Meta, StoryObj } from '@storybook/angular';
import { argsToTemplate, moduleMetadata } from '@storybook/angular';
import {MatProgressSpinnerModule,MatProgressSpinner} from '@angular/material/progress-spinner';

const meta: Meta<MatProgressSpinner> = {
  title: 'Components/Progress Spinner',
  component: MatProgressSpinner,
  tags: ['autodocs'],
  argTypes: {
    color: {
      control: 'radio',
      options: ['accent', 'primary', 'warn'],
      description: 'Theme color palette for the component.',
      table: {
        category: 'INPUTS',
        defaultValue: { summary: '-' },
      },
    },
    diameter: {
      control: 'number',
      description:
        'The diameter of the progress spinner (will set width and height of svg).',
      table: {
        category: 'INPUTS',
        defaultValue: { summary: '-' },
      },
    },
    mode: {
      control: 'radio',
      options: ['determinate', 'indeterminate', 'buffer', 'query'],
      description:
        'Mode of the progress bar.Input must be one of these values: determinate, indeterminate, buffer, query, defaults to determinate.Mirrored to mode attribute.',
      table: {
        category: 'INPUTS',
        defaultValue: { summary: 'determinate' },
      },
    },
    strokeWidth: {
      control: 'number',
      description: 'Stroke width of the progress spinner.',
      table: {
        category: 'INPUTS',
        defaultValue: { summary: '-' },
      },
    },
    value: {
      control: 'number',
      description:
        'Value of the progress bar. Defaults to zero. Mirrored to aria-valuenow.',
      table: {
        category: 'INPUTS',
        defaultValue: { summary: '-' },
      },
    },
    defaultColor: {
      control: 'string',
      description: 'Default color to fall back to if no value is set.',
      table: {
        category: 'INPUTS',
        defaultValue: { summary: '-' },
      },
    },
  },
  decorators: [
    moduleMetadata({
      imports: [MatProgressSpinnerModule],
    }),
  ],
  render: (args: MatProgressSpinner) => ({
    props: args,
    template: `<mat-progress-spinner aria-label="spinner" ${argsToTemplate(
      args
    )}> </mat-progress-spinner>
    `,
  }),
};

export default meta;
type Story = StoryObj<MatProgressSpinner>;

export const Default: Story = {
  args: {
    color: 'primary',
    mode: 'indeterminate',
    diameter: 80,
    strokeWidth: 5,
  },
};

export const Warn: Story = {
  args: {
    ...Default.args,
    color: 'warn',
  },
};

export const Accent: Story = {
  args: {
    ...Default.args,
    color: 'accent',
  },
};

export const StrokeWidth: Story = {
  args: {
    ...Default.args,
    strokeWidth:10,
  }
};

export const Diameter: Story = {
  args: {
    ...Default.args,
    strokeWidth:10,
    diameter:100
  }
};