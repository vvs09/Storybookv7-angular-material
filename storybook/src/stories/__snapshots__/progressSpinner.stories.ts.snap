// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Progress Spinner Accent smoke-test 1`] = `
<storybook-root ng-version="15.2.10">
  <mat-progress-spinner role="progressbar"
                        tabindex="-1"
                        aria-label="spinner"
                        class="mat-mdc-progress-spinner mdc-circular-progress mat-accent mdc-circular-progress--indeterminate"
                        ng-reflect-color="accent"
                        ng-reflect-mode="indeterminate"
                        ng-reflect-diameter="80"
                        ng-reflect-stroke-width="5"
                        aria-valuemin="0"
                        aria-valuemax="100"
                        mode="indeterminate"
                        style="width: 80px; height: 80px;"
  >
    <div aria-hidden="true"
         class="mdc-circular-progress__determinate-container"
    >
      <svg xmlns="http://www.w3.org/2000/svg"
           focusable="false"
           class="mdc-circular-progress__determinate-circle-graphic"
           viewbox="0 0 75 75"
      >
        <circle cx="50%"
                cy="50%"
                class="mdc-circular-progress__determinate-circle"
                r="35"
                style="stroke-dasharray: 219.911px; stroke-width: 6.25%;"
        >
        </circle>
      </svg>
    </div>
    <div aria-hidden="true"
         class="mdc-circular-progress__indeterminate-container"
    >
      <div class="mdc-circular-progress__spinner-layer">
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-left">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__gap-patch">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-right">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
      </div>
    </div>
  </mat-progress-spinner>
</storybook-root>
`;

exports[`Components/Progress Spinner Default smoke-test 1`] = `
<storybook-root ng-version="15.2.10">
  <mat-progress-spinner role="progressbar"
                        tabindex="-1"
                        aria-label="spinner"
                        class="mat-mdc-progress-spinner mdc-circular-progress mat-primary mdc-circular-progress--indeterminate"
                        ng-reflect-color="primary"
                        ng-reflect-mode="indeterminate"
                        ng-reflect-diameter="80"
                        ng-reflect-stroke-width="5"
                        aria-valuemin="0"
                        aria-valuemax="100"
                        mode="indeterminate"
                        style="width: 80px; height: 80px;"
  >
    <div aria-hidden="true"
         class="mdc-circular-progress__determinate-container"
    >
      <svg xmlns="http://www.w3.org/2000/svg"
           focusable="false"
           class="mdc-circular-progress__determinate-circle-graphic"
           viewbox="0 0 75 75"
      >
        <circle cx="50%"
                cy="50%"
                class="mdc-circular-progress__determinate-circle"
                r="35"
                style="stroke-dasharray: 219.911px; stroke-width: 6.25%;"
        >
        </circle>
      </svg>
    </div>
    <div aria-hidden="true"
         class="mdc-circular-progress__indeterminate-container"
    >
      <div class="mdc-circular-progress__spinner-layer">
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-left">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__gap-patch">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-right">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
      </div>
    </div>
  </mat-progress-spinner>
</storybook-root>
`;

exports[`Components/Progress Spinner Diameter smoke-test 1`] = `
<storybook-root ng-version="15.2.10">
  <mat-progress-spinner role="progressbar"
                        tabindex="-1"
                        aria-label="spinner"
                        class="mat-mdc-progress-spinner mdc-circular-progress mat-primary mdc-circular-progress--indeterminate"
                        ng-reflect-color="primary"
                        ng-reflect-mode="indeterminate"
                        ng-reflect-diameter="100"
                        ng-reflect-stroke-width="10"
                        aria-valuemin="0"
                        aria-valuemax="100"
                        mode="indeterminate"
                        style="width: 100px; height: 100px;"
  >
    <div aria-hidden="true"
         class="mdc-circular-progress__determinate-container"
    >
      <svg xmlns="http://www.w3.org/2000/svg"
           focusable="false"
           class="mdc-circular-progress__determinate-circle-graphic"
           viewbox="0 0 100 100"
      >
        <circle cx="50%"
                cy="50%"
                class="mdc-circular-progress__determinate-circle"
                r="45"
                style="stroke-dasharray: 282.743px; stroke-width: 10%;"
        >
        </circle>
      </svg>
    </div>
    <div aria-hidden="true"
         class="mdc-circular-progress__indeterminate-container"
    >
      <div class="mdc-circular-progress__spinner-layer">
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-left">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 100 100"
          >
            <circle cx="50%"
                    cy="50%"
                    r="45"
                    style="stroke-dasharray: 282.743px; stroke-dashoffset: 141.372px; stroke-width: 10%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__gap-patch">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 100 100"
          >
            <circle cx="50%"
                    cy="50%"
                    r="45"
                    style="stroke-dasharray: 282.743px; stroke-dashoffset: 141.372px; stroke-width: 10%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-right">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 100 100"
          >
            <circle cx="50%"
                    cy="50%"
                    r="45"
                    style="stroke-dasharray: 282.743px; stroke-dashoffset: 141.372px; stroke-width: 10%;"
            >
            </circle>
          </svg>
        </div>
      </div>
    </div>
  </mat-progress-spinner>
</storybook-root>
`;

exports[`Components/Progress Spinner StrokeWidth smoke-test 1`] = `
<storybook-root ng-version="15.2.10">
  <mat-progress-spinner role="progressbar"
                        tabindex="-1"
                        aria-label="spinner"
                        class="mat-mdc-progress-spinner mdc-circular-progress mat-primary mdc-circular-progress--indeterminate"
                        ng-reflect-color="primary"
                        ng-reflect-mode="indeterminate"
                        ng-reflect-diameter="80"
                        ng-reflect-stroke-width="10"
                        aria-valuemin="0"
                        aria-valuemax="100"
                        mode="indeterminate"
                        style="width: 80px; height: 80px;"
  >
    <div aria-hidden="true"
         class="mdc-circular-progress__determinate-container"
    >
      <svg xmlns="http://www.w3.org/2000/svg"
           focusable="false"
           class="mdc-circular-progress__determinate-circle-graphic"
           viewbox="0 0 80 80"
      >
        <circle cx="50%"
                cy="50%"
                class="mdc-circular-progress__determinate-circle"
                r="35"
                style="stroke-dasharray: 219.911px; stroke-width: 12.5%;"
        >
        </circle>
      </svg>
    </div>
    <div aria-hidden="true"
         class="mdc-circular-progress__indeterminate-container"
    >
      <div class="mdc-circular-progress__spinner-layer">
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-left">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 80 80"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 12.5%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__gap-patch">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 80 80"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 12.5%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-right">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 80 80"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 12.5%;"
            >
            </circle>
          </svg>
        </div>
      </div>
    </div>
  </mat-progress-spinner>
</storybook-root>
`;

exports[`Components/Progress Spinner Warn smoke-test 1`] = `
<storybook-root ng-version="15.2.10">
  <mat-progress-spinner role="progressbar"
                        tabindex="-1"
                        aria-label="spinner"
                        class="mat-mdc-progress-spinner mdc-circular-progress mat-warn mdc-circular-progress--indeterminate"
                        ng-reflect-color="warn"
                        ng-reflect-mode="indeterminate"
                        ng-reflect-diameter="80"
                        ng-reflect-stroke-width="5"
                        aria-valuemin="0"
                        aria-valuemax="100"
                        mode="indeterminate"
                        style="width: 80px; height: 80px;"
  >
    <div aria-hidden="true"
         class="mdc-circular-progress__determinate-container"
    >
      <svg xmlns="http://www.w3.org/2000/svg"
           focusable="false"
           class="mdc-circular-progress__determinate-circle-graphic"
           viewbox="0 0 75 75"
      >
        <circle cx="50%"
                cy="50%"
                class="mdc-circular-progress__determinate-circle"
                r="35"
                style="stroke-dasharray: 219.911px; stroke-width: 6.25%;"
        >
        </circle>
      </svg>
    </div>
    <div aria-hidden="true"
         class="mdc-circular-progress__indeterminate-container"
    >
      <div class="mdc-circular-progress__spinner-layer">
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-left">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__gap-patch">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
        <div class="mdc-circular-progress__circle-clipper mdc-circular-progress__circle-right">
          <svg xmlns="http://www.w3.org/2000/svg"
               focusable="false"
               class="mdc-circular-progress__indeterminate-circle-graphic"
               viewbox="0 0 75 75"
          >
            <circle cx="50%"
                    cy="50%"
                    r="35"
                    style="stroke-dasharray: 219.911px; stroke-dashoffset: 109.956px; stroke-width: 6.25%;"
            >
            </circle>
          </svg>
        </div>
      </div>
    </div>
  </mat-progress-spinner>
</storybook-root>
`;
